core/store/migrations/migration1580827546/migrate.go

```diff
@@ -0,0 +1,13 @@
+package migration1580827546
+
+import (
+	"github.com/jinzhu/gorm"
+)
+
+// Migrate adds the polling_interval duration (nanoseconds) to support the Flux Monitor.
+func Migrate(tx *gorm.DB) error {
+	return tx.Exec(`
+	  ALTER TABLE job_runs RENAME COLUMN overrides TO initial_params;
+	`).Error
```

Comment:
So, our strategy for Chainlink is to have zero down time.  From what I understand this means no renaming or removing column names.  Is this correct @j16r?  Any suggestion on how to deal with this?

Reply:
Yeah, a rename on a column causes a full table lock, which will cause significant downtime for our app. Especially considering the job_runs tables have historically been quite large.

Reply:
Don't migrations only run on app startup?

Reply:
@samphilipd that is correct.

Reply:
If we go the route of copying to another column, I think we'd said previously that this column might make more sense on the RunRequest table.